SPECS/python-google-pasta/python-google-pasta.spec

```diff
@@ -0,0 +1,51 @@
+Summary:        Enable python source code refactoring through AST modifications
+Name:           python-google-pasta
+Version:        0.2.0
+Release:        1%{?dist}
+License:        ASLv2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Development/Languages/Python
+URL:            https://github.com/google/pasta/
+Source0:        https://github.com/google/pasta/archive/v%{version}.tar.gz#/google-pasta-%{version}.tar.gz
+BuildArch:      noarch
+BuildRequires:  python3-devel
+%if %{with_check}
+BuildRequires:  python3-pip
+%endif
```

Comment:
Double-checking: is `python3-pip` needed for the tests in this case? We're not using it in the `%check` section explicitly, but I don't what can be expected of `setup.py`.

Reply:
I will remove the pip from the build requirement and do the build to see if it fails or not.